models/dvmdostem/inst/dvmdostem.pecan.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0"?>
```

Comment:
If this fill is just for your own personal testing, it probably doesn't have to be committed. If it's a more general test, it should be in the tests directory. That said, this specific file won't run since it lacks run, site, and input information.